scripts/build-site.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+export NODE_ENV="production"
+
+# We use the Git SHA to name our CSS and JS bundles uniquely. In most cases, we'll get the
+# SHA from GitHub Actions, but in case we don't, just fall back to Git.
+export GIT_SHA=${GITHUB_SHA:=$(git rev-list HEAD | head -1)}
+
+# Paths to the CSS and JS bundles we'll generate below. Note that environment variables
+# are read by some templates during the Hugo build process.
+export CSS_BUNDLE="public/css/styles.${GIT_SHA}.css"
+export JS_BUNDLE="public/js/bundle.min.${GIT_SHA}.js"
+
+printf "Copying prebuilt docs...\n\n"
+make copy_static_prebuilt
+
+printf "Building web components...\n\n"
+make build_components
+
+printf "Running Hugo...\n\n"
+hugo --minify "$@" --templateMetrics -e production
+
+printf "Compiling the JavaScripts...\n\n"
+yarn run tsc --outFile ${JS_BUNDLE}
```

Comment:
Does this mean we're no longer minifying? Not sure it matters one way or another.

Reply:
Ah yes, that's right, although that wasn't intentional, although gzipped, the payload is still about the same as today, so I kind of agree it's probably not worth addressing yet.